src/Core/Form/IdentifiableObject/DataFormatter/BulkCombinationFormDataFormatter.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataFormatter;
+
+class BulkCombinationFormDataFormatter
+{
+    /**
+     * @param array<string, mixed> $formData
+     *
+     * @return array<string, mixed>
+     */
+    public function format(array $formData): array
+    {
+        $formData['references']['reference'] = $formData['reference'];
```

Comment:
Should we be prudent about bad form structure and verify offsets exist?

Reply:
Probably.
But this will change a little, I need to test it manually and make sure all fields reaches this code part empty if no data was changed. 
Null in data provider won't probably solve it because it will be considered as empty value by symfony form (and will arrive here with values `0` or `''`).